borg/helpers.py

```diff
@@ -212,7 +212,11 @@ def show_progress(self, item=None, final=False, stream=None, dt=None):
 
 def get_keys_dir():
     """Determine where to repository keys and cache"""
-    xdg_config = os.environ.get('XDG_CONFIG_HOME', os.path.join(os.path.expanduser('~'), '.config'))
+    if os.environ.get('HOME'):
+        home_dir = os.environ.get('HOME')
+    else:
+        home_dir = os.path.expanduser('~%s' % os.environ.get('USER'))
```

Comment:
if USER is not in environment, the .get() will return None. So I think this rather should be .get('USER', '').